Automattic/syntaxhighlighter
# Migrate project to wp-scripts and fix lodash deps

Fixes #267 

### Changes proposed in this Pull Request

* Migrate project from cgb-scripts to wp-scripts
* Fix lodash deps

### Testing instructions

* Checkout the project
* `npm run build`
* Create a new post and add a syntaxhighlighter/code block
* ✅ Should work
* Play with it a bit and check preview
* ✅ Should work
* Install the plugin from the generated zip file and do the same test procedure
* ✅ Should work
